phoenix-core/src/main/java/org/apache/phoenix/compile/DeleteCompiler.java

```diff
@@ -354,7 +354,11 @@ private List<PTable> getClientSideMaintainedIndexes(TableRef tableRef) {
         }
         return Collections.emptyList();
     }
-    
+
+    /**
+     * Implementation of MutationPlan that is selected if there is no index presented for a table
+     * and no LIMIT clause in the DELETE statement.
+     */
     public class MultiRowDeleteMutationPlan implements MutationPlan {
```

Comment:
let's also add that this plan gets selected if the query is strictly point lookup?

Reply:
`the query is strictly point lookup` => `the query either has no WHERE clause or it is strictly point lookup`